linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/rule.yml

```diff
@@ -72,3 +72,11 @@ ocil: |-
     functionality completely.
     If the option is set to a number greater than <tt>0</tt>, then the session will be disconnected after
     <tt>ClientAliveInterval * ClientAliveCountMax</tt> seconds without receiving a keep alive message.
+
+template:
+    name: sshd_lineinfile
+    vars:
+        parameter: ClientAliveCountMax
+        rule_id: sshd_set_keepalive
```

Comment:
I found it weird that the rule_id is passed as a parameter here because the rule ID is known to the templating engine.

Reply:
I confirm it is not used. I fixed it in the sshd_set_keepalive. I also made a commit which removes it from other templated rules, just in case it would feel too much out of scope here.